.travis.yml

```diff
@@ -14,8 +14,10 @@ before_script:
   - GO_FILES=$(find . -iname '*.go' | grep -v /vendor/)  # All the .go files, excluding vendor/ if any
   - PKGS=$(go list ./... | grep -v /vendor/)             # All the import paths, excluding vendor/ if any
   - curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh # Install latest dep release
+  - go get github.com/rakyll/embedmd
 
 script:
+  - embedmd -d README.md
```

Comment:
The output of this command will not be committed back to the repo.

Remove this step from travis.

Reply:
actually the intention here is not to commit it back to the repo but to verify that there are no differences between what would be generated and what is in the repo (the command returns non-zero upon finding a difference). I will add a comment to this effect.